docusaurus.config.js

```diff
@@ -77,6 +77,45 @@ module.exports = {
         {
           type: "docsVersionDropdown",
           position: "left",
+          dropdownItemsAfter: [
+            {
+              type: "html",
+              value: '<hr class="dropdown-separator">',
+            },
+            {
+              type: "html",
+              className: "dropdown-archived-versions",
+              value: "<b>Archived versions</b>",
```

Comment:
Internally, I've asked PM for feedback on what to call this. I'll also follow up in the C8 Stakeholder meeting.